### DISCUSSION TITLE
Many-to-Many associations and junction table

### DISCUSSION THREAD

**Discussion by jsanguin** [sequelize/sequelize#15148] [Help & Questions]
The documentation states that when a string is provided in the through option, Sequelize will create the joint table. At what point this table will be created? Once you tried to add something to it? Thanks and regards.

**Comment by ephys** [comment]
The documentation saying it will create the through table is a bit of a misnomer. It's creating the through model (if it doesn't already exist) (which it does immediately)

It only creates the table if (and when) you call `sequelize.sync()`
